skuba-update/VERSION

```diff
@@ -0,0 +1 @@
+1.2.6
```

Comment:
Do we need to version control this file?

Reply:
It's probably better, but it's not required, you're right! Fair point!

If it's version controlled, it just makes it simpler in the package building I guess: It won't require a `make` to build, as all the artifacts will already be present.

Reply:
You're probably right, it's maybe even less error prone to not have it versioned (what if someone doesn't pull the last tags?).

In that case, does the pre-commit hook still make sense?

I am puzzled, on the fence between the two implementations. I like the readability of the file being present, and therefore no generation needs to happen in rpm building. But I realise the errors the developers might see.

Reply:
Keeping it in version control means it will be easy to unintentionally commit it since it will always be changing when developers run `make`. The version number should only be updated when the release manager actually wants to make a release.

Reply:
The version file should be updated by release engineer, so the pre-commit hook is no longer needed :smile:

Reply:
> Keeping it in version control means it will be easy to unintentionally commit it since it will always be changing when developers run `make`. The version number should only be updated when the release manager actually wants to make a release.

Wouldn't it be better in this case to have it version controlled, and _remove_ all the make wiring?

Wondering what's the right thing here, because having it _not version controlled_ means reintroducing the change in ``ci/packaging/suse/skuba_spec_template``, which makes it less obvious to track.

IMO it's not only when release needs to happen, but when at development time, we are ready to say "This deserves a bump in the release version".

I am definitely on the fence.